server/cmwell-cons/src/main/scala/ctl.scala

```diff
@@ -2005,6 +2009,8 @@ abstract class Host(user: String,
               withUpdateSchemas: Boolean = false,
               hosts: GenSeq[String] = ips) {
 
+    val currentVersion = extractVersionFromProcNode(ips(0))
+    currentVersion.map(ver => info(s"Current version is $ver"))
```

Comment:
What if we failed to fetch /proc/node or the JSON parsing failed for some reason?
We will just move on with upgrade. I recommend in this case to use `Await.result` even it's not considered as a best practice, we don't want to proceed with upgrade if we don't know the current version. 
(In future we will refactor the whole thing to be less imperative.)